projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -25,9 +25,47 @@ import {
 } from './navigation';
 import { generateMail, randomString } from './user';
 
-export function addCartForB2BCustomer(): void {
-  const productCode = '1979039';
-  cy.login('gi.sun@pronto-hw.com', 'pw4all').then(() => {
+const asmForB2CCustomer = 'aaron.customer@hybris.com';
+const asmForB2BCustomer = 'Gi Sun';
+
+export function placeOrderForB2CCustomer(
+  customer: string,
+  pwd: string,
+  productCode: string
+): void {
+  cy.login(customer, pwd).then(() => {
+    const auth = JSON.parse(localStorage.getItem('spartacus⚿⚿auth'));
```

Comment:
LGTM except the key icon is strange. Just make sure the tests can pass. I will approve the PR.

Reply:
All tests local tests passed